src/components/Nav/Nav.scss

```
@@ -0,0 +1,121 @@
+@mixin li-default {
+  padding: 20px;
+  float: left;
+  color: #1c1c1c;
+  font-size: 13px;
+  list-style: none;
+}
+@mixin navstyle {
+  width: 100%;
+  padding-bottom: 20px;
+  background-color: #fdf9ee;
+}
+@mixin hover {
+  font-weight: bold;
+  text-decoration: underline;
+}
+
+.menu {
+  margin-top: 17px;
+  margin-left: 40px;
+  padding-bottom: 30px;
+  float: left;
+  color: #1c1c1c;
+  font-size: 13px;
+}
+
+.logo {
+  display: block;
+  width: 140px;
+  height: 50px;
+  margin: 0 auto;
+}
+.navigate {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 65px;
+  padding-top:10px;
+  background-color: #fdf9ee;
+  li:hover{
+    @include hover;
+  }
+}
+.icons {
+  margin-right: 30px;
+  span {
+    margin-top: 8px;
+    margin-right: 20px;
+    color: #1c1c1c;
+    font-size: 28px;
+    font-weight: 200;
+  }
+}
+.inactive{
+  display: none;
+}
+.hovernav {
+  @include navstyle;
+  .secondmenulist{
+    display: inline-block;
+    margin-left: 20px;
+    .secondmenu{
+      @include li-default
+    }
```

```suggestion
.hovernav {
  @include navstyle;
  
  .secondmenulist{
    display: inline-block;
    margin-left: 20px;
  
      .secondmenu{
      @include li-default
    }
```
스타일 간격의 가독성을 위해 한칸씩 띄워주세요!